src/_P037_MQTTImport.ino

```diff
@@ -761,9 +761,10 @@ bool MQTTSubscribe_037(struct EventStruct *event)
   for (uint8_t x = 0; x < VARS_PER_TASK; x++) {
     String subscribeTo = P037_data->getFullMQTTTopic(x);
 
-    if (!subscribeTo.isEmpty()) {
-      parseSystemVariables(subscribeTo, false);
+    parseSystemVariables(subscribeTo, false);
+    subscribeTo.trim();
 
+    if (!subscribeTo.isEmpty()) {
```

Comment:
Ah that was a silly mistake....

Reply:
I guess in a common use-case it wasn't supposed to be empty 😃

Reply:
Yep, sometimes we're just like AI... copy/paste/paste/paste ;)